docs/api.md

```diff
@@ -127,6 +127,111 @@ Possible error code responses. New error codes may be added in future releases.
 | `hmac_invalid`          | 400         | No    | The `ekeyhmac` field, when base64 decoded is not the right size (32 bytes) |
 |                         | 500         | Yes   | Internal processing error, may be successful on retry. |
 
+# Admin APIs
+
+These APIs are available on the admin server and require and `ADMIN` level API key.
+
+## `/api/issue`
+
+Request a verification code to be issued. Accepts [optional] symptom date and test dates in ISO 8601 format. These can be in local time, if a timezone offset is provided. If a phone number is provided and the realm is configured with SMS credentials, then an SMS will be dispatched according to the realm's settings.
+
+```json
+{
+	"symptomDate": "YYYY-MM-DD",
+	"testDate": "YYYY-MM-DD",
+	"testType": "<valid test type>",
+	"tzOffset": 0,
+	"phone": "+1NPANXXLINE",
+}
+```
+
+* `sypmtomDate` and `testDate` are both optional.
+  * only one will be encoded into the eventually issued certificate
+  * symptom date is always preferred to test date
+* `testType`
+  * Must be `confirmed`, `likely`, `negative`
+  * valid values depends on your realm's settings
+* `tzOffset`
+  * Offset in minutes of the user's timezone. Positive, negative, 0, or omitted (using the default of 0) are all valid. 0 is considered to be UTC.
```

Comment:
Minutes? Isn't this usually hours?

Reply:
this is the way it is in the api - not all TZ offsets are even hour increments